app/gui/src/model/module/synchronized.rs

```diff
@@ -256,29 +256,32 @@ impl API for Module {
 // === Synchronizing Language Server ===
 
 impl Module {
-    /// The asynchronous task scheduled during struct creation which listens for all module changes
-    /// and send proper updates to Language Server.
-    async fn runner(
+    /// Returns the asynchronous task scheduled during struct creation which listens for all module
+    /// changes and send proper updates to Language Server.
```

Comment:
I think it suggests that the "scheduling during struct creation" is somehow a part of this function.
```suggestion
    /// Returns the asynchronous task which listens for all module
    /// changes and send proper updates to Language Server.
```